molecule/resources/tasks/create_network.yml

```diff
@@ -0,0 +1,27 @@
+---
+# Create network in gcp project, a subnet and a firewall rule.
+
+- name: "Create network {{ molecule_yml.driver.network_name }}"
+  gcp_compute_network:
+    name: "{{ molecule_yml.driver.network_name }}"
+    state: present
+    project: "{{ molecule_yml.driver.gcp_project_id }}"
+    auth_kind: serviceaccount
+    service_account_file: "{{ molecule_yml.driver.gcp_service_account_key }}"
+  register: network
+
+- name: "Create firewall rule for ports 22, 3389 and 5986"
```

Comment:
ports 3389 and 5986 are unnecessary (for now) aren't they? They're only needed for Windows.

Reply:
Re windows: Windows machines need to run the following script https://github.com/ansible/ansible/blob/devel/examples/scripts/ConfigureRemotingForAnsible.ps1  before Ansible is able to connect to them. I'm not sure of a way to do running that script other than manually logging into the machines. Is it acceptable to have some dedicated windows machines with the script installed running in our project?

Reply:
well this part of the PR was reverted anyway, but I guess I would say that I would have put the ports 3389 and 5986 into the same PR that turns on windows testing. Or is windows already enabled somehow and I'm just not seeing it?

Regardless, it doesn't matter because you got rid of the create network step.

Reply:
No you're correct, im planning to add Windows tests in a follow-up PR. I was just raising the issue now that because of the unusual setup required we can't initialize windows machines on the fly, they'd require dedicated instances.